contentcuration/contentcuration/utils/assessment/qti/base.py

```
@@ -0,0 +1,283 @@
+                            if current_children:
+                                current_children[-1].tail = (
+                                    current_children[-1].tail or ""
+                                ) + item.text
+                            else:
+                                element.text = (element.text or "") + item.text
+
+                    continue
+                else:
+                    raise ValueError(
+                        "List types should only contain XMLElement or TextNodes"
+                    )
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "continue", remove the "else" and de-indent the code inside it

(R1724)